src/controllers/adminController.ts

```diff
@@ -1,7 +1,9 @@
 import express, { Request, Response } from "express";
+import dotenv from "dotenv";
 import { registrar } from "../services/registrar";
 import { adminMiddleware } from "../middleware/adminMiddleware";
 import logger from "../services/logger";
+dotenv.config({ path: "./.env.local" });
```

Comment:
This looks like the wrong place to have this

Reply:
oops, didn't actually need that. removed it